stubs/flake8-bugbear/METADATA.toml

```
@@ -0,0 +1 @@
+version = "21.11.29"
```

Maybe this should be:

```suggestion
version = "21.11.*"
```